src/app/navbar/navbar.controller.ts

```diff
@@ -52,6 +57,12 @@ export class CheNavBarController {
         this.gotoProfile();
       }
     },
+    {
+      name: 'Copy Login Command',
```

Comment:
From the sprint review:

> @benoitf : Maybe say ‘chectl login command’ as I may think it’s an oc or kubectl command (but I’m not a UX expert)

I'm OK with either option. We made similar as OpenShift Console 3.x has.

Reply:
from UX: `Copy crwctl Login Command`/`Copy crwctl Login Command`

Reply:
there is one that should be chectl no ?

Reply:
sure, that's just a typo